### PR TITLE
Style (Cf 347): Move graph actions to its own section

### PR DESCRIPTION
**Pull request by kellieyau**
Moved tools to the bottom left:
image

### CODE REVIEW

**Comment by kellieyau** [comment]
added more margin space cause it was too close, the new one is:
image

File: frontend/src/pages/GraphicalSelector/styles.ts [hunk 1]
```diff
@@ -32,9 +32,16 @@ const SearchBarWrapper = styled.div`
   right: 20px;
 `;
 
+const ToolsWrapper = styled.div`
+  display: flex;
+  align-items: flex-end;
```
Comment (gtangelo) [inline frontend/src/pages/GraphicalSelector/styles.ts]: do
```
align-items: center;
gap: 10px;
```

**Review by gtangelo — CHANGES_REQUESTED** [review]

File: frontend/src/pages/GraphicalSelector/styles.ts (hunk 1 above)
Comment (kellieyau) [reply frontend/src/pages/GraphicalSelector/styles.ts]: new look: image

Comment (gtangelo) [reply frontend/src/pages/GraphicalSelector/styles.ts]: Oh lol I meant replace the `align-items: flex-end` line with the above 😅. So keep in the display, margin and other css styles haha

Comment (kellieyau) [reply frontend/src/pages/GraphicalSelector/styles.ts]: oh lol ok sweet cause i was gonna be like looks a bit dodge rn
xD

Comment (kellieyau) [reply frontend/src/pages/GraphicalSelector/styles.ts]: anyway should be fixed with hte new stuff: 
image

**Review by gtangelo — APPROVED** [review]

**Pull request merged**